labscript_devices/SpinnakerCamera/blacs_workers.py

```diff
@@ -198,12 +198,14 @@ def configure_acquisition(self, continuous=True, bufferCount=10):
             self.set_stream_attribute('StreamBufferHandlingMode', 'NewestFirst')
             self.set_attribute('AcquisitionMode', 'Continuous')
         elif bufferCount == 1:
-            self.set_stream_attribute('StreamBufferCountMode', 'Auto')
-            self.set_stream_attribute('StreamBufferHandlingMode', 'OldestFirst')
+            self.set_stream_attribute('StreamBufferCountMode', 'Manual')
+            self.set_stream_attribute('StreamBufferCountManual', 3)
```

Comment:
I'm struggling to find in the documentation where the minimum number has to be 3. Best I can find is that `OldestFirstOverwrite` needs it since it requires 2 frames more than the expected number. A single frame in `NewestFirst`/`OldestFirst` should only need 2 (expected frame +1). 

It's not so much that I doubt you, more that their documentation is terrible.